src/wifi/AdafruitIO_MKR1010.h

```diff
@@ -0,0 +1,81 @@
+/*!
+ * @file AdafruitIO_MKR1010.h
+ *
+ * Adafruit invests time and resources providing this open source code.
+ * Please support Adafruit and open source hardware by purchasing
+ * products from Adafruit!
+ *
+ * Copyright (c) 2015-2016 Adafruit Industries
+ * Authors: David Goldstein, Morgan Winters
+ * Licensed under the MIT license.
+ *
+ * All text above must be included in any redistribution.
+ */
+
```

Comment:
Please add an include guard here to avoid the WINC1500 header file from being imported.

something like,
```
#ifndef ADAFRUITIO_MKR1010_H
#define ADAFRUITIO_MKR1010_H
```